index.html

```diff
@@ -265,6 +265,42 @@ <h3>CSS</h3>
 
   </div>
 
-  
+  <footer class="contact" id="Contact">
+    <div class="contact-elements">
+      <div class="content">
+        <h2>Contact me</h2>
+          <p>
+          If you have an application you are
+         interested in developing, a
+         feature that you need built or
+         a project that needs coding.
+        I’d love to help with it 
+          </p>
+      </div>
+      <form id="contact-form" action="https://formspree.io/f/xwkykkjv" method="post">
+        <ul>
+          <li>
+            <label for="name">Name</label>
+            <input type="text" id="name" name="name" required maxlength="30" placeholder="Your Name">
+          </li>
+          <li>
+            <label for="mail">Email</label>
+            <input type="email" id="mail" name="email" required placeholder="youremail@mail.com">
+          </li>
+          <li>
+            <label for="message">Message</label>
+            <textarea id="message" name="message" required maxlength="500" placeholder="Write your message here"></textarea>
+          </li>
+          <li>
+            <button id="submit" type="submit" class="cta-button">
+              Get in touch
+            </button>
+          </li>
+        </ul>
+      </form>
```

Comment:
- [ ] Kindly remove the labels so your design matches the Figma design
- Your design:
<img width="212" alt="image" src="https://user-images.githubusercontent.com/77190203/154298242-9910b6cf-be14-4372-a090-7a6eb02af5d1.png">
- Figma design:
<img width="151" alt="image" src="https://user-images.githubusercontent.com/77190203/154298420-2e066b5e-1807-4f87-a6f2-52ca818cf0b4.png">

Reply:
Done